applications/crossbar/src/modules/cb_local_provisioner_templates.erl

```diff
@@ -66,22 +67,23 @@ init() ->
 %% @end
 %%--------------------------------------------------------------------
 -spec content_types_provided(cb_context:context(), path_token(), path_token()) -> cb_context:context().
-content_types_provided(#cb_context{req_verb = ?HTTP_GET, auth_account_id=AccountId}=Context, DocId, ?IMAGE_REQ) ->
-    Db = wh_util:format_account_id(AccountId, 'encoded'),
+-spec content_types_provided(cb_context:context(), http_method(), path_token(), path_token()) -> cb_context:context().
+content_types_provided(Context, PT1, PT2) ->
+    content_types_provided(Context, cb_context:req_verb(Context), PT1, PT2).
```

Comment:
I need a rolled up newspaper for you and @macpie; swat you with it each time you do this.